dwh/indicators.py

```diff
@@ -109,8 +124,69 @@ def find_date_range(args: argparse.Namespace) -> Tuple[str, str, str]:
       patient_agg_obs_df, ARV_plan=_ARV_PLAN,
       start_drug=[_DRUG1], end_date_str=end_date)
 
+  TB_STAT_df = indicator_lib.calc_TB_STAT(
+      patient_agg_obs_df, TB_TX_plan=_TB_TX_PLAN, ARV_plan=_ARV_PLAN,
+      TB_plan_answer=[_START_DRUG], end_date_str=end_date)
+
+  TX_CURR_df = indicator_lib.calc_TX_CURR(
+      patient_agg_obs_df, ARV_plan=_ARV_PLAN,
+      ARV_plan_answer=[_CONTINUE_REGIMEN, _START_DRUG],
+      end_date_str=end_date)
+
+  TB_ART_df = indicator_lib.calc_TB_ART(
+      patient_agg_obs_df, TB_TX_plan=_TB_TX_PLAN, ARV_plan=_ARV_PLAN,
+      TB_plan_answer=[_CONTINUE_REGIMEN, _START_DRUG],
+      ART_plan_answer=[_CONTINUE_REGIMEN, _START_DRUG],
+      end_date_str=end_date)
+
+  TB_PREV_df = indicator_lib.calc_TB_PREV(
+      patient_agg_obs_df, TB_PREV_plan=_TB_PREV_plan, ARV_plan=_ARV_PLAN,
+      TB_PREV_plan_answer=[_CONTINUE_REGIMEN, _START_DRUG],
+      TB_CURR_plan_answer=[_COMPLETE_REGIMEN],
+      ART_plan_answer=[_CONTINUE_REGIMEN, _START_DRUG],
+      end_date_str=end_date)
+
+  # Because we can't do muliple constraints for one code
+  base_query = query_lib.patient_query_factory(
+      query_lib.Runner.SPARK, args.src_dir, _CODE_SYSTEM)
+  tb_annual_query = base_query.include_obs_values_in_time_range(
```

Comment:
The returned object is the same; simply rename `base_query` to `semiannual_query` or something like that and ignore the returned value here (`self` is returned for easy chaining).